app/models/asset.rb

```diff
@@ -24,6 +24,12 @@ class Asset < ApplicationRecord
   }.freeze
 
   NON_SUPPORT_ATTRIBUTES = SUPPORT_ATTRIBUTES.except(:department_sold_to_id, :location_cc_ship_to_account).freeze
+  CLOSURE_EMAILING_ATTRIBUTES = {
```

Comment:
`EMAILING` feels very specific. What about something more abstract like `NOTIFICATION`?

Reply:
Nice. I will rename it